### PR TITLE
Hosted Exchange Mailbox Download & FAQ

### PR DESCRIPTION
**Pull request by Rohi2305**

### CODE REVIEW

File: content/how-to/exchange/hosted_exchange_mailbox/index.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
+---
+permalink: hosted_exchange_mailbox
```
Comment (ssalinas24) [inline content/how-to/exchange/hosted_exchange_mailbox/index.md:2]: I noticed in the other article we used underscore too. I would suggest we change to a dash instead here.  so `hosted-exchange-mailbox`. Rename the folder as well. Update any links or permalinks that point to this file as well.

File: content/how-to/exchange/hosted_exchange_mailbox/index.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+audit_date: '2022-12-17'
+title: Hosted Exchange Mailbox Download & FAQ
+type: article
+created_date: '2022-12-17'
+created_by: Rohit Rajput
```
Comment (ssalinas24) [inline content/how-to/exchange/hosted_exchange_mailbox/index.md:7]: If this is the value that shows on the page as author, then put `Tom Mitchell`

File: content/how-to/exchange/hosted_exchange_mailbox/index.md [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+
+**i**.  In the top navigation bar, click **Select a Product>>Dedicated Hosting**.
+
+**ii**.	Click **Products>>Cloud Office**.
+
+**iii**. In the Cloud Office window, click **Open Cloud Office Control Panel**.
+
+**Step 2.** Click **Manage** drop-down menu, next to each mailbox and select **Download Mailbox**. In the Download Mailbox window, enable checkbox to accept the terms and click **Download Mailbox button.**
+
+<img width="468" alt="manage_download_mailbox" src="manage_download_mailbox.png">
```
Comment (ssalinas24) [inline content/how-to/exchange/hosted_exchange_mailbox/index.md:49]: Look at my other PR and you will notice that I just removed the width attribute entirely on each image. This lets the image be whatever size it was saved as. It showed also automatically shrink down smaller when you shrink the browser window and the image should be responsive. If so, then remove the width here and on the others in this file as well.

Comment (ssalinas24) [reply content/how-to/exchange/hosted_exchange_mailbox/index.md:49]: Fix alt tag

File: content/how-to/exchange/hosted_exchange_mailbox/index.md [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+
+**Step 1**. Log in to [Manage Office 365 at Rackspace Control Panel](https://manage365.rackspace.com/login?ReturnUrl=%2F) with the account's **Global Admin** credentials. 
+
+**Step 2**. In the home page, if the mailbox data is ready for download, click the **Mail Recovery** button to view the Mail Recovery download page.
+
+<img width="307" alt="mail_recovery" src="mail_recovery.png">
+
+**Note:** If the mailbox data is not ready for download, the home page will display the following message requesting to check back later.
+
+<img width="250" alt="email_note1" src="https://user-images.githubusercontent.com/96761530/208292027-5995310e-c0db-4f09-8ad9-e9c775e43853.png">
```
Comment (ssalinas24) [inline content/how-to/exchange/hosted_exchange_mailbox/index.md:62]: take care of the width, alt tag, and source path

File: content/how-to/exchange/hosted_exchange_mailbox/index.md [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+
+**Step 3**. The list of Hosted Exchange mailboxes will appear on the Mail Recovery page. You can also search for the users based on name. Click the **Download .pst** link next to each mailbox to download that user's PST file.
+
+<img width="308" alt="download_pst" src="https://user-images.githubusercontent.com/96761530/208293182-7f240315-5a2c-42c2-b9b3-d8c2c066bb58.png">
```
Comment (ssalinas24) [inline content/how-to/exchange/hosted_exchange_mailbox/index.md:66]: take care of the width, alt tag, and source path

File: content/how-to/exchange/hosted_exchange_mailbox/index.md [hunk 6]
```diff
@@ -0,0 +1,170 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (ssalinas24) [inline content/how-to/exchange/hosted_exchange_mailbox/index.md:170]: Remove all these extra lines

**Review by ssalinas24 — COMMENTED** [review]
@Rohi2305 I have left comments below with lessons learned from the previous PR and changes that you should make.
Once you get the updated final word doc from stakeholders for this article, then you can review the text and images line by line to see what they changed. You can have your word doc side by side next to your browser. Just make browser smaller so you can read them next to each other.

Then make the changes and just try to keep whatever text they gave you and don't try to change any of the text so we don't alter the meaning of anything accidentally.

Once you make the changes, then you can send the deploy-preview link to Tom and Vanessa. Ask if they need any other changes or if you are good to Publish. Once they agree to publish then you can either merge the PR or have Lane merge it. Worst case you can't get a hold of him, then go ahead and ping me.

**Comment by lfielder** [comment]
Already duplicated and completed the article creation

**Pull request closed without merging**